haas/api.py

```diff
@@ -927,23 +883,22 @@ def stop_console(nodename):
     ####################
 
 
-def _assert_absent(session, cls, name):
+def _assert_absent(cls, name):
```

Comment:
Maybe document that this function, and the following ones, require you to be in request context